lib/testLauncher/composeConfig.js

```diff
@@ -22,28 +22,27 @@ const HOME_DIR = IS_WIN
 	? process.env.USERPROFILE
 	: process.env.HOME;
 
-const CONFIG_FILES = [
-	`.${APP_NAME}rc.js`,
-	`.${APP_NAME}rc.json`,
-	`.${APP_NAME}rc.yaml`,
-	`.${APP_NAME}rc.yml`,
-	`.${APP_NAME}rc.json5`,
-	`.${APP_NAME}rc.ini`,
-	`.${APP_NAME}rc`,
-];
-
 /**
- * @description should repeat lookup logic from rc library
- * @type {string[]}
+ * @description default directories to search. Logic the same as in rc
+ * @type {
+	* {
+	* path: string,
+	* filename: string,
+	* deepSearch: boolean,
+	* isWin: boolean
+	* } []
+ * }
  */
-const LOOKUP_DIRS = [
-	path.join(HOME_DIR, `.${APP_NAME}rc`),
-	path.join(HOME_DIR, `.${APP_NAME}`, 'config'),
-	path.join(HOME_DIR, '.config', APP_NAME),
-	path.join(HOME_DIR, '.config', APP_NAME, 'config'),
-	path.join(ETC_DIR, `.${APP_NAME}rc`),
-	path.join(ETC_DIR, APP_NAME, 'config'),
-	process.cwd(),
+const DEFAULT_PATHS = [
+	{path: process.cwd(), filename: `.${APP_NAME}rc`, deepSearch: true, isWin: true},
+	{path: path.join(HOME_DIR, '.config', APP_NAME), filename: 'config', deepSearch: false, isWin: true},
+	{path: path.join(HOME_DIR, '.config'), filename: APP_NAME, deepSearch: false, isWin: true},
+	{path: path.join(HOME_DIR, `.${APP_NAME}`), filename: 'config', deepSearch: false, isWin: true},
+	{path: HOME_DIR, filename: `.${APP_NAME}rc`, deepSearch: false, isWin: true},
+
+	{path: process.cwd(), filename: `.${APP_NAME}rc`, deepSearch: true, isWin: false},
```

Comment:
Seemds this line is duplicate of first line because of `isWin` prop? Could you pls get rid of duplication?